src/test/java/microsoft/exchange/webservices/data/util/TimeZoneUtilsTest.java

```diff
@@ -49,12 +49,15 @@ public void testGetMicrosoftTimeZoneNameBad() {
     // null-argument is not allowed.
     try {
       Assert.fail(TimeZoneUtils.getMicrosoftTimeZoneName(null));
-    } catch (final IllegalArgumentException ignored) {}
-
-    // Case-insensitive ID is not supported.
-    checkGetMicrosoftTimeZoneName("africa/abidjan", "UTC");
+    } catch (final IllegalArgumentException ignored) {
+    }
   }
 
+  @Test
+  public void testCaseInsensitiveIdNotSupported() {
+    // Case-insensitive ID is not supported so the map look up returns null.
+    checkGetMicrosoftTimeZoneName("africa/abidjan", null);
```

Comment:
I actually dont see what this test is about? Do you want to test `TimeZone.getTimeZone(...)` ? Since this is a java.util and no ews-java-api class testing if it will return null on case-insensitive is not related to our project. This test should be removed.